### ISSUE TITLE
Always returns wrong design - Avoid this plugin

### ISSUE THREAD

**Issue by ciarans** [bliblidotcom/vue-rangedate-picker#72]

**Comment by jonnylink** [comment]
Yeah, looks nice, but it is pretty buggy. Even the demo doesn't work.

**Comment by abennett2** [comment]
If you view the demo, the calendar on has 30 days for May and only 29 for June. 

I came here to report that when I tested locally, the start date came back EDT and the end date came back EST. Not sure why.

Growing concerned I'm going to need to use a different plugin

**Comment by ciarans** [comment]
@abennett2 we opted for this in the end https://github.com/ankurk91/vue-flatpickr-component
